CMakeLists.txt

```diff
@@ -55,7 +55,7 @@
 cmake_minimum_required (VERSION 2.8 FATAL_ERROR)
 
 project(SLiM)
-
+include(ExternalProject)
```

Comment:
Does this have any side effects worth worrying about?  Can it be made to happen only on Windows?

Reply:
I can make it happen only on Windows to be safe.